### PR TITLE
Add Dtos for Client and PetSitter endpoints, AB#46

### PR DESCRIPTION
**Pull request by Lukas-Stanleyy**
Please check the Dtos.

### CODE REVIEW

File: backend/PetSupport.API/Dtos/OutDtos/BookingMessageFullDTO.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+using System;
+using PetSupport.Data.Entities;
+
+namespace PetSupport.API.Dtos.OutDtos
+{
+    public class BookingMessageFullDTO : BaseEntity
+    {
+        public int Id { get; set; }
+        public DateTime SendDate { get; set; }
+        public string Message { get; set; }
```
Comment (zdulak) [inline backend/PetSupport.API/Dtos/OutDtos/BookingMessageFullDTO.cs:10]: Change Client type to ClientDto and create adequate ClientDto type

File: backend/PetSupport.API/Dtos/InDtos/CreatePetsitterDTO.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        public string Surname { get; set; }
+        public string Email { get; set; }
+        public string PhoneNumber { get; set; }
+        public string ImageId { get; set; }
+        public string Description { get; set; }
+        
+        public string Address { get; set; }
+        public string City { get; set; }
+        public string ZipCode { get; set; }
+
```
Comment (zdulak) [inline backend/PetSupport.API/Dtos/InDtos/CreatePetsitterDTO.cs:18]: Good idea in todo

**Review by zdulak — CHANGES_REQUESTED** [review]
Add FindPetSitterLongForm and FindPetSitterShortForm dtos

File: backend/PetSupport.API/Dtos/OutDtos/BookingMessageFullDTO.cs (hunk 1 above)
Comment (Lukas-Stanleyy) [reply backend/PetSupport.API/Dtos/OutDtos/BookingMessageFullDTO.cs:10]: Done

**Pull request closed without merging**